meta-resin-common/recipes-devtools/go-cross/go-cross_1.6.bbappend

```diff
@@ -1 +1,2 @@
 DEPENDS_append = " virtual/${TARGET_PREFIX}gcc libgcc"
+INHIBIT_SYSROOT_STRIP = "1"
```

Comment:
The reason behind this is simply, this is how meta-oe configures a similar package  (i,e. gcc-cross). I keep it for consistency + because this is a host tool and stripping does not provide an advantage.